.travis.yml

```diff
@@ -1,10 +1,10 @@
 language: elixir
-elixir:
-  - 1.1.0
-  - 1.2.0
-otp_release:
-  - 17.4
-  - 18.1
+elixir: 1.4.0
+otp_release: 19.2
+matrix:
+  include:
+    - elixir: 1.1.0
```

Comment:
We don't need to worry about 1.1 anymore but I think we should probably add coverage for 1.3 and 1.2, what do you think?